impl/src/main/resources/application.yaml

```diff
@@ -3,6 +3,9 @@ nuts:
     schedule:
       delay: 15 * 60 * 1000
       initial_delay: 1000
+  event:
+    meta:
+      location: "."
```

Comment:
not in a ./data dir or something?

Shouldn't this be configurable with env vars e.g., 
```
location: ${EVENT_META_LOCATION:.}
```

Reply:
yes, see consent-bridge.rst